sdk/storage/azblob/blockblob/models.go

```diff
@@ -23,6 +23,14 @@ type Block = generated.Block
 // BlockList - can be uncommitted or committed blocks (committed/uncommitted)
 type BlockList = generated.BlockList
 
+// BlobCopySourceTags - can be 'COPY' or 'REPLACE'
+type BlobCopySourceTags = generated.BlobCopySourceTags
+
+const (
+	BlobCopySourceTagsCopy    = generated.BlobCopySourceTagsCOPY
+	BlobCopySourceTagsReplace = generated.BlobCopySourceTagsREPLACE
+)
```

Comment:
This part should also be moved to `blockblob/contants.go`.

Reply:
Done!